src/Symfony/Bundle/FrameworkBundle/Resources/config/property_access.xml

```diff
@@ -4,10 +4,22 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
+    <parameters>
+        <parameter key="property_accessor.cache.prefix" />
+    </parameters>
+
     <services>
         <service id="property_accessor" class="Symfony\Component\PropertyAccess\PropertyAccessor" >
             <argument /> <!-- magicCall, set by the extension -->
             <argument /> <!-- throwExceptionOnInvalidIndex, set by the extension -->
+            <argument /> <!-- cache, set by the extension -->
+        </service>
+
+        <!-- Cache -->
+        <service id="property_accessor.cache.apc" class="Doctrine\Common\Cache\ApcCache" public="false">
```

Comment:
I've looked more in depth at `Doctrine\Cache\PhpFileCache` and it does a lot of FS access. It's not adapted for this kind of cache. Finally APC/APCu is still the best available option and we should not deprecate/remove it.